keps/20200225-reading-parameter-values-from-a-file.md

```diff
@@ -0,0 +1,120 @@
+---
+kep-number: draft-20200225
+title: Reading parameter values from a file
+short-desc: Reading parameter values from a file
+authors:
+  - "@porridge"
+owners:
+  - "@porridge"
+editor: @porridge
+creation-date: 2020-02-25
+last-updated: 2020-02-25
+status: provisional
+see-also:
+  - KEP-24
+  - KEP-25
```

Comment:
I don't see how kep-25 template toyaml is associated

Reply:
As explained by @ANeumann82 above: the motivation for this KEP is to defines a mechanism to inject values that will later be consumed by `toYaml`, therefore it needs to consider the typical use cases of `toYaml` to be useful.

Reply:
Added a mention to `Motivation` section.